### PR TITLE
🔀🌐 [ko_KR] add translation for ko_KR.ts

### PR DESCRIPTION
**Pull request by sheisfly**
**What kind of update does this PR provide?** *Please check*

- [x] new translations / updated translations / translation fixes
- [ ] a new feature
- [ ] a new module
- [ ] a bugfix for an existing feature / module
- [ ] improvement of style or user experience
- [ ] other: *Please fill out*

**Which language(s) did you add/update translations for?**
ko_KR

**What is included in your update?**
* new translations for vehicles
* new translations for vehicle categories
* new translations for building categories
* new translations for schoolings
**Additional notes**
Please add any further notes here!

### CODE REVIEW

File: src/i18n/ko_KR.ts [hunk 1]
```diff
@@ -0,0 +1,669 @@
...
+    pois: [
+        '공원',
+        '호수',
+        '병원',
+        '숲',
+        '버스 정류장',
+        '전차 정류장',
+        '기차역(지역 교통)',
+        '기차역(지역 교통 및 장거리 여행역)',
+        '화물역',</select        '
```
Comment (jxn-30) [inline src/i18n/ko_KR.ts:579]: There is some additionaly code, accidentally.

**Review by jxn-30 — CHANGES_REQUESTED** [review]

File: src/i18n/ko_KR.ts (hunk 1 above)
Comment (sheisfly) [reply src/i18n/ko_KR.ts]: I modified it

**Comment by jxn-30** [comment]
@LSS-Manager deploy

File: src/i18n/ko_KR.ts (hunk 1 above)
Comment (Cr4zyc4k3) [reply src/i18n/ko_KR.ts]: No way that `</select` is a proper POI in korea

File: src/i18n/ko_KR/buildings.ts [hunk 2]
```diff
@@ -0,0 +1,961 @@
...
+        caption: 'Staging area',
+        color: '#c259b5',
+        coins: 0,
+        credits: 0,
+        extensions: [],
+        levelcost: [],
+        maxBuildings: 4,
+        maxLevel: 0,
+        special:
+            'You can station as many of your own vehicles as you like at a staging area, members of the association can use the room. A staging area remains for 24 hours, but you can reset it to 24 hours at any time.With Premium Account you can have 8 staging areas at the same time',
```
Comment (Cr4zyc4k3) [inline src/i18n/ko_KR/buildings.ts:406]: `time.With` => `time. With`

**Review by Cr4zyc4k3 — APPROVED** [review]
Please remove the typo and this PR is ready for merging.

**Comment by kdev** [comment]
This PR is stale. Do we want to implement the translations manually without waiting for @sheisfly to fix the typos? This is also the only change that needs to be made which is holding this whole translation up.

**Review by Cr4zyc4k3 — APPROVED** [review]

**Comment by Cr4zyc4k3** [comment]
@LSS-Manager deploy

**Comment by Cr4zyc4k3** [comment]
It seems like that this is not the only issue. Would vote on closing.

**Comment by jxn-30** [comment]
closing here due to too much unfinished translation & being stale
I don't see how LSSM should work very much even if the Issue of latest build would be fixed.

**Pull request closed without merging**